deployments/upgrades/export_configs_to_helm.py

```
@@ -0,0 +1,339 @@
+}
+
+# Fields on roles that are internal metadata, not config.
+ROLE_INTERNAL_FIELDS = {
+    'sync_mode',
+}
+
+
+def fetch(base_url, path, headers):
+    """Fetch JSON from the OSMO API."""
+    url = f'{base_url}{path}'
+    req = urllib.request.Request(url, headers=headers)
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)